.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve the project.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Bug Description
+A clear and concise description of what the bug is.
+
+### How to Reproduce
+Steps to reproduce the behaviour:
+1. Go to '...'
+2. Execute code '...'
+3. See error
+
+### Expected behaviour
+A clear and concise description of what you expected to happen.
+
+### Further Information
```

Comment:
I would really like to alter this section in contrast to the [oshdb](https://github.com/GIScience/oshdb/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) a little bit. It would be nice to have some kind of template with _endpoint_, _parameters_, _data_(?), _used requesting tool/library_ … in it to remind people that it's easier to bugfix if we have the important information.